home/migrations/0002_add_groups.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models, migrations
+from home.models.models import Group
+from django.conf import settings
+
+
+def add_groups(apps, schema_editor):
+    Group.objects.get_or_create(name="Staff"),
+    Group.objects.get_or_create(name="Student"),
+    Group.objects.get_or_create(name="Teacher"),
+    Group.objects.get_or_create(name="Volunteer"),
+    Group.objects.get_or_create(name="Donor"),
```

Comment:
The commas at the ends of the lines shouldn't be here, and the names should be lowercase (per how they're used elsewhere in the code)

Reply:
This was fixed